### DISCUSSION TITLE
set sam cli to use more of width for output logs about CloudFormation

### DISCUSSION THREAD

**Discussion by liesislukas** [aws/aws-cli#6957] [Q&A] [answered]
There is plenty of space on the right yet CloudFormation logs are always squeezed in small space and hard to read. Here is short sample of wrapping text, in long log readability becomes real issue.

![image](https://user-images.githubusercontent.com/2733862/168649722-5aae1dbf-1515-4115-b763-7216144a7cb7.png)

Some other log messages do use a lot of width, it's just CloudFormation table. I assume it "counts" available space and uses that, yet it's running inside Code Build instance, so there is no display, which ends up messing available space calculations. 

Is there any config option to set width of this table?

**Comment by tim-finnigan** [comment, marked as answer]
Hi @liesislukas I think this is intended for the SAM CLI repo: https://github.com/aws/aws-sam-cli/issues
